package.json

```diff
@@ -44,7 +44,6 @@
 	"scripts": {
 		"preanalyze-bundles": "cross-env-shell CALYPSO_ENV=production EMIT_STATS=true PROGRESS=1 NODE_ARGS=--max_old_space_size=8192 npm run -s build-client-evergreen",
 		"analyze-bundles": "webpack-bundle-analyzer stats.json public/evergreen -h 127.0.0.1 -p 9898 -s gzip",
```

Comment:
Won't this line need the same fix as line 115?

Reply:
> Won't this line need the same fix as line 115?

Yes, I pushed a fix for `analyze-bundles`, too.